nibabel/volumeutils.py

```diff
@@ -504,7 +507,8 @@ def array_from_file(shape, in_dtype, infile, offset=0, order='F', mmap=True):
             pass
     if len(shape) == 0:
         return np.array([])
-    n_bytes = int(np.prod(shape) * in_dtype.itemsize)
```

Comment:
Fair enough. Might change the comment to reflect that it's not just avoiding int32 overflows, but that's really nit-picky.